test/kafka_client_SUITE.erl

```diff
@@ -65,8 +65,8 @@ produce(Config) ->
                       Key,
                       M),
 
-    ?assertMatch({ok, [#kafka_message{key=Key,
-                                      value=M} | _]}, brod:fetch(Hosts, Topic, 0, 0)),
+    %% ?assertMatch({ok, [#kafka_message{key=Key,
```

Comment:
should this be uncommented?

Reply:
No, brod doesn't support this kafka. We can only test the metadata requests with brod right now.